### PR TITLE
Project - Jieun Park

### PR DESCRIPTION
**Pull request by CSSjieun**

### CODE REVIEW

**Comment by CSSjieun** [comment]
final version is done, ready for review.

File: _projects/2023/100509696/100509696.Rmd [hunk 1]
```diff
@@ -0,0 +1,170 @@
+---
+title: "New dataviz project"
```
Comment (Enchufa2) [inline _projects/2023/100509696/100509696.Rmd]: Could you please provide a descriptive title?

File: _projects/2023/100509696/100509696.Rmd [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+description: |
+  A short description of the post.
```
Comment (Enchufa2) [inline _projects/2023/100509696/100509696.Rmd]: And a descriptive summary about what's the graph about.

File: _projects/2023/100509696/100509696.Rmd [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+```
+
+This graph explains how the electricity generation sources have been changed in the United States from 2001 to 2017 over 16 years. The data covers all 50 states from the United States. Sources are arranged in order of percentages and it has crossover features in the graph. 
+
+Usually, over 16 years, coal, nuclear, and natural gas are the main resources of electric generation in the U.S. According to the article, the ranking of different sources have not been changed very much almost over 17 years for 13 states. In this other 37 states, both the percentages of electricity generated and the relative ordering for different sources have changed noticeably. Therefore, the crossovers of different sources for the electiricty generation in the graph are affected mostly by these 37 states. 
+
+## Original graph
+```{r}
+knitr::include_graphics("~/Desktop/data visualization/ElectricitySourcesLN-superJumbo.webp")
```
Comment (Enchufa2) [inline _projects/2023/100509696/100509696.Rmd]: Please copy this file next to this Rmd and commit and push it.

File: _projects/2023/100509696/100509696.Rmd [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+```
+
+## Data and Data Processing
+
+```{r}
+data <- read_excel("~/Desktop/data visualization/Net_generation_United_States_all_sectors_annual.xls")
```
Comment (Enchufa2) [inline _projects/2023/100509696/100509696.Rmd]: The same for the data. All datasets used here should be put in this directory and pushed as part of the PR.

File: _projects/2023/100509696/100509696.Rmd [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+  geom_sankey_bump(space = 0, type = "alluvial", color = "transparent", smooth = 13, alpha = 0.93) +
+  theme_sankey_bump(base_size = 7) +
+  scale_fill_manual(values = color)
+```
+
+## Enhancement
+
+```{r}
+
+```
```
Comment (Enchufa2) [inline _projects/2023/100509696/100509696.Rmd]: Something is missing here?

**Review by Enchufa2 — CHANGES_REQUESTED** [review]
Thanks, I left some comments below. Please address them and let me know when this is ready again.

**Comment by CSSjieun** [comment]
I changed the things as comment. Could you please check it?

**Comment by CSSjieun** [comment]
Just in case I added the data in the same directory right next to the Rmd. file and commit and pushed it again.

**Review by Enchufa2 — APPROVED** [review]
Neat! Merging with a few fixes.

**Comment by Enchufa2** [comment]
Here's your work, congrats! https://csslab.uc3m.es/dataviz/projects/2023/100509696/

**Pull request merged**